src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const fixedRobot = {};
+
+  for (const keys in robot) {
```

Comment:
it's one key, why plural?
```suggestion
  for (const key in robot) {
```